sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractRulesDefinitionTest.java

```diff
@@ -51,17 +56,27 @@ public void test() {
 
   @Test
   public void test_before_9_3() {
-    SonarRuntime sonarRuntime = SonarRuntimeImpl.forSonarQube(Version.create(9, 2), SonarQubeSide.SCANNER, SonarEdition.COMMUNITY);
-    AbstractRulesDefinition sut = new TestRulesDefinition(sonarRuntime);
-    RulesDefinition.Context context = new RulesDefinition.Context();
-    sut.define(context);
+    assertThat(getSecurityStandards(Version.create(9, 2), "S1111")).containsExactlyInAnyOrder("cwe:117", "cwe:532", "owaspTop10:a10", "owaspTop10:a3");
+  }
 
-    RulesDefinition.Repository repository = context.repository("test");
-    assertThat(repository).isNotNull();
+  @Test
+  public void test_security_standards_9_4_PCI_DSS_is_not_available() {
+    assertThat(getSecurityStandards(Version.create(9, 4), PCI_DSS_RULE_KEY)).containsExactlyInAnyOrder();
```

Comment:
This assertion is not clear. It should be something like `isEmpty`, shouldn't it?

Reply:
I meant the assertion, not the method name
```suggestion
    assertThat(getSecurityStandards(Version.create(9, 4), PCI_DSS_RULE_KEY)).isEmpty();
```